### PR TITLE
Add UUID column to JsonBlobs

### PR DESCRIPTION
**Pull request by TomTirapani**
See hoist-react PR: https://github.com/xh/hoist-react/pull/2102

### CODE REVIEW

File: grails-app/domain/io/xh/hoist/jsonblob/JsonBlob.groovy [hunk 1]
```diff
@@ -40,7 +42,7 @@ class JsonBlob implements JSONFormat {
     }
 
     Map formatForJSON() {[
-        id: id,
+        id: uuid,
```
Comment (haynesjm42) [inline grails-app/domain/io/xh/hoist/jsonblob/JsonBlob.groovy:45]: Not sure we want to conflate the id/uuid here. I think we are fine to still return the id just as we do for other domain objects along with the new uuid

Comment (lbwexler) [reply grails-app/domain/io/xh/hoist/jsonblob/JsonBlob.groovy:45]: I agree with this -- thought we would call the new thing token?

Comment (TomTirapani) [reply grails-app/domain/io/xh/hoist/jsonblob/JsonBlob.groovy:45]: The name 'token' came later, as a result of john's review. I'm in the process of updating this branch

File: grails-app/domain/io/xh/hoist/jsonblob/JsonBlob.groovy [hunk 2]
```diff
@@ -42,7 +42,8 @@ class JsonBlob implements JSONFormat {
     }
```
Comment (lbwexler) [inline grails-app/domain/io/xh/hoist/jsonblob/JsonBlob.groovy]: no need/value to add a unique constraint to token?

**Pull request merged**